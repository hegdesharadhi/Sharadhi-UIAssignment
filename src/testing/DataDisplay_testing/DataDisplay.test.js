import { render, screen } from '@testing-library/react'
import { data } from './mockData'
import DataDisplay from '../../components/DataDisplay'

describe('DataDisplay', () => {
  test('Check the DataDisplay when data is empty', () => {
    render(<DataDisplay data={data} />)
    const loading = screen.getByText(/Loading/i)
    expect(loading).toMatchSnapshot()
  })
})
