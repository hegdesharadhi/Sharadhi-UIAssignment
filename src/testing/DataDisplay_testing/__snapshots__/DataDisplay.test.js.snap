// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataDisplay Check the DataDisplay when data is empty 1`] = `
<div>
  Loading...
</div>
`;
